sda/cmd/api/api_test.go

```diff
@@ -710,6 +710,37 @@ func testEndpoint(c *gin.Context) {
 	c.JSON(200, gin.H{"ok": true})
 }
 
+func (s *TestSuite) TestLogLevelDebug() {
+	// Capture log output from gin in buf
+	var buf bytes.Buffer
+	log.SetOutput(&buf)
+	// Ensure we log to stdout after this test is completed
+	defer log.SetOutput(os.Stdout)
+
+	log.SetLevel(log.DebugLevel)
+	srv := setup(Conf)
+	srv.Addr = ":8080" // TODO: What is the proper way to configure this from api_test.go?
+	go func() {
+		if err := srv.ListenAndServe(); err != nil {
+			log.Fatal(err)
```

Comment:
```suggestion
			s.FailNow("failed to start server")
```

Never use `log.Fatal` or similar things in tests.
When a test fails we want to know why so using the built in `Fail/FailNow` functions is much better.